app/utils/routeConstants.js

```diff
@@ -6,5 +6,9 @@ export default {
       padding: 20
     },
     exact: true
+  },
+  floating: {
+    route: '/floating',
+    props: {}
```

Comment:
we use yarn instead of npm, so we already have a yarn.lock which contains the dependency versions. 

please use yarn and delete the package-lock.json